### DISCUSSION TITLE
i can't export .mp3

### DISCUSSION THREAD

**Discussion by WorkEveryday1** [mifi/lossless-cut#2081] [General]
Error: Command failed with exit code 1: C:\Users\Az\Desktop\LosslessCut-win-x64\resources\ffmpeg.exe -hide_banner -i C:\Users\Az\Desktop\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700.mp4 -map 0:0 -c:0 copy -map 0:1 -c:1 copy -map_metadata 0 -movflags +faststart -default_mode infer_no_subs -ignore_unknown -f mp3 -y C:\Users\Az\Desktop\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700-00.00.00.000-00.03.33.716.mp3
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'C:\Users\Az\Desktop\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    creation_time   : 2024-07-20T13:31:29.000000Z
    encoder         : Lavf60.3.100
  Duration: 00:03:33.72, start: 0.000000, bitrate: 215 kb/s
  Stream #0:0[0x1](und): Video: h264 (Main) (avc1 / 0x31637661), yuv420p(progressive), 1280x720 [SAR 1:1 DAR 16:9], 77 kb/s, 30 fps, 30 tbr, 30004 tbn (default)
    Metadata:
      creation_time   : 2024-07-20T13:31:29.000000Z
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
      encoder         : AVC Coding
  Stream #0:1[0x2](und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 131 kb/s (default)
    Metadata:
      creation_time   : 2024-07-20T13:31:29.000000Z
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
[mp3 @ 000001e2adc3b500] Invalid audio stream. Exactly one MP3 audio stream is required.
[out#0/mp3 @ 000001e2adf13500] Could not write header (incorrect codec parameters ?): Invalid argument
[aost#0:1/copy @ 000001e2ad93e9c0] Error initializing output stream: 
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
    Last message repeated 1 times

    at makeError (C:\Users\Az\Desktop\LosslessCut-win-x64\resources\app.asar\out\main\index.js:1800:13)
    at handlePromise (C:\Users\Az\Desktop\LosslessCut-win-x64\resources\app.asar\out\main\index.js:2619:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

{
  "err": {
    "killed": false,
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "exitCode": 1
  },
  "state": {
    "ffmpegExperimental": false,
    "preserveMovData": false,
    "movFastStart": true,
    "preserveMetadataOnMerge": false,
    "filePath": "C:\\Users\\Az\\Desktop\\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700.mp4",
    "fileFormat": "mp3",
    "externalFilesMeta": {},
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "h264",
        "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
        "profile": "Main",
        "codec_type": "video",
        "codec_tag_string": "avc1",
        "codec_tag": "0x31637661",
        "width": 1280,
        "height": 720,
        "coded_width": 1280,
        "coded_height": 720,
        "closed_captions": 0,
        "film_grain": 0,
        "has_b_frames": 0,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "16:9",
        "pix_fmt": "yuv420p",
        "level": 32,
        "chroma_location": "left",
        "field_order": "progressive",
        "refs": 1,
        "is_avc": "true",
        "nal_length_size": "4",
        "id": "0x1",
        "r_frame_rate": "30/1",
        "avg_frame_rate": "192385648/6411945",
        "time_base": "1/30004",
        "start_pts": 30,
        "start_time": "0.001000",
        "duration_ts": 6411945,
        "duration": "213.703006",
        "bit_rate": "77566",
        "bits_per_raw_sample": "8",
        "nb_frames": "6412",
        "extradata_size": 38,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "creation_time": "2024-07-20T13:31:29.000000Z",
          "language": "und",
          "handler_name": "VideoHandler",
          "vendor_id": "[0][0][0][0]",
          "encoder": "AVC Coding"
        }
      },
      {
        "index": 1,
        "codec_name": "aac",
        "codec_long_name": "AAC (Advanced Audio Coding)",
        "profile": "LC",
        "codec_type": "audio",
        "codec_tag_string": "mp4a",
        "codec_tag": "0x6134706d",
        "sample_fmt": "fltp",
        "sample_rate": "44100",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "initial_padding": 0,
        "id": "0x2",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/44100",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 9424896,
        "duration": "213.716463",
        "bit_rate": "131331",
        "nb_frames": "9204",
        "extradata_size": 2,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "creation_time": "2024-07-20T13:31:29.000000Z",
          "language": "und",
          "handler_name": "SoundHandler",
          "vendor_id": "[0][0][0][0]"
        }
      }
    ],
    "copyStreamIdsByFile": {
      "C:\\Users\\Az\\Desktop\\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700.mp4": {
        "0": true,
        "1": true
      }
    },
    "cutSegments": [
      {}
    ],
    "mainFileFormatData": {
      "filename": "C:\\Users\\Az\\Desktop\\NONSTOP VINAHOUSE 2023 - STEREO LOVE REMIX - MY LOVE REMIX - DON'T YA PHIÊN BẢN BASS DOWN LOW TIKTOK - Trim-00.00.01.433-00.03.34.700.mp4",
      "nb_streams": 2,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "213.716463",
      "size": "5764375",
      "bit_rate": "215776",
      "probe_score": 100,
      "tags": {
        "major_brand": "isom",
        "minor_version": "512",
        "compatible_brands": "isomiso2avc1mp41",
        "creation_time": "2024-07-20T13:31:29.000000Z",
        "encoder": "Lavf60.3.100"
      }
    },
    "rotation": 360,
    "shortestFlag": false,
    "effectiveExportMode": "separate",
    "outSegTemplate": "${FILENAME}-${CUT_FROM}-${CUT_TO}${SEG_SUFFIX}${EXT}"
  },
  "platform": "win32",
  "version": "3.61.1"
}

**Comment by mifi** [comment]
it appears you are trying to mux an mp3 into an mp4, which is not technically possible. try select output format as `mp3` instead.
